docs/Feature Documentation/Audio and Video Streaming Setup/index.md

```diff
@@ -170,33 +115,33 @@ gst-launch-1.0 souphttpsrc location=http://127.0.0.1:5050 ! "application/x-rtp-s
 gst-launch-1.0 souphttpsrc location=http://127.0.0.1:5080 ! audio/x-raw,format=S16LE,rate=16000,channels=1 ! pulsesink
 ```
 
-# Start SDL Core
-In the build folder
+## Video Streaming States
 
-```
-cd bin
-./start.sh
-```
-!!! NOTE
-If you want to use a USB connection, you must run
-```
-sudo ./start.sh
-```
-!!!
+This section describes how Core manages the streaming states of mobile applications. Only one application may stream video at a time, but audio applications may stream while in the LIMITED state with other applications.
+
+When an app is moved to HMI level `FULL`:
+- All non-streaming applications go to HMI level `BACKGROUND`
+- All apps with the same App HMI Type go to `BACKGROUND`
+- Streaming apps with a different App HMI Type that were in `FULL` go to `LIMITED`
+
+When an app is moved to HMI level `LIMITED`:
+- All non-streaming applications keep their HMI level
+- All applications with a different App HMI Type keep their HMI level
+- Applications with the same App HMI Type go to `BACKGROUND`
 
-### Start the web HMI
+### Start the Web HMI
 
 CD into the HMI repository and run
 ```
 chromium-browser index.html
 ```
 
-### Start Video or Audio Stream
+### Additional Resources
 
 !!! NOTE
-No open source mobile application currently implements video streaming.
+Livio provides an [example video streaming android application](https://github.com/livio/sdl_video_streaming_android_sample).
 !!!
 
-[iOS Video Streaming Guide](../../iOS/mobile-navigation/video-streaming/)
+[iOS Video Streaming Guide](../../../iOS/video-streaming-for-navigation-apps/video-streaming/)
```

Comment:
This link doesn't work when viewing it in a PR. I think it should just be a static link 
```suggestion
[iOS Video Streaming Guide](https://smartdevicelink.com/en/guides/iOS/video-streaming-for-navigation-apps/video-streaming/)
```

Reply:
on PR it adds two more levels (/pull_request and /$hash) so I tested the links adding two more `../` to the relative paths and they worked fine.

Reply:
I'm viewing it in with the `6.1 Release Overhaul Testing` version. I think the link should be fixed so it can work consistently, regardless of whether it's in a PR or in production

Reply:
But a full path will not work properly in staging... it's a trade off either way but I think a relative path is the correct thing to do here, so any of the server config can change and it can still work or so we can host a development version (staging) of the guides. I really think using a full path in a situation like this is just the lazy option. If you're sure the correct trade off is to use a full path let me know and I'll change it.

Reply:
hmm though decision but I think for now for the sake of consistency and allowing it to work in PRs I think the full link might be better for now

Reply:
added in https://github.com/smartdevicelink/sdl_core_guides/pull/71/commits/9f48ec382fbbe03a31807656c63517457e3cf268